main.go

```diff
@@ -165,6 +185,11 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "Upgrade")
 		os.Exit(1)
 	}
+
+	if os.Getenv("ENABLE_WEBHOOKS") != "true" || setupWebhookControllers(mgr) != nil {
```

Comment:
nit: it is hard to read this construction, I think it is better to use nested if conditions.

Reply:
Done